docs/source/api_reference/layers.rst

```diff
@@ -207,6 +207,9 @@ Torch Layers
 .. autoclass:: deepchem.models.torch_models.layers.DTNNEmbedding
   :members:
 
+.. autoclass:: deepchem.models.torch_models.layers.EdgeNetwork
```

Comment:
Make sure to add to https://github.com/deepchem/deepchem/blob/master/docs/source/api_reference/torch_layers.csv as well

Reply:
Yes sure, I'll do.